diff --git a/krb/files/etc/krb5kdc/kdc.conf b/krb/files/etc/krb5kdc/kdc.conf
index a3ab394..27234d9 100644
--- a/krb/files/etc/krb5kdc/kdc.conf
+++ b/krb/files/etc/krb5kdc/kdc.conf
@@ -2,7 +2,7 @@
 	kdc_ports = 750,88
 
 [realms]
-	RSYSLOG3 = {
+	RSYSLOG3m = {
 		database_name = /var/lib/krb5kdc/principal
 		admin_keytab = FILE:/etc/krb5kdc/kadm5.keytab
 		acl_file = /etc/krb5kdc/kadm5.acl
diff --git a/krb/files/kadminhttp.py b/krb/files/kadminhttp.py
index 324cfc8..9b9b31d 100755
--- a/krb/files/kadminhttp.py
+++ b/krb/files/kadminhttp.py
@@ -167,7 +167,7 @@ if __name__=="__main__":
 	parser = argparse.ArgumentParser()
 	parser.add_argument("--bind", default="0.0.0.0", help="bind address")
 	parser.add_argument("--port", default=47900, type=int, help="bind address")
-	parser.add_argument("--realm", default="RSYSLOG3", help="bind address")
+	parser.add_argument("--realm", default="RSYSLOG3m", help="bind address")
 	
 	parser.add_argument("--debug", action='store_true', default=False, help="debug")
 
diff --git a/krb/templates/etc/krb5.conf.erb b/krb/templates/etc/krb5.conf.erb
index 7ac603a..2dbd02f 100644
--- a/krb/templates/etc/krb5.conf.erb
+++ b/krb/templates/etc/krb5.conf.erb
@@ -1,5 +1,5 @@
 [libdefaults]
-	default_realm = RSYSLOG3
+	default_realm = RSYSLOG3m
 	kdc_timesync = 1
 	ccache_type = 4
 	forwardable = true
@@ -9,13 +9,13 @@
 	allow_weak_crypto = true
 
 [realms]
-	RSYSLOG3 = {
+	RSYSLOG3m = {
 		kdc = <%= @kdc_server_real %>
 		admin_server = <%= @kdc_server_real %>
-		default_domain = meta.zcu.cz
 	}
 
 [domain_realm]
-	.meta.zcu.cz = RSYSLOG3
-	meta.zcu.cz = RSYSLOG3
-	
+	A = RSYSLOG3h
+	B = RSYSLOG3h
+	C = RSYSLOG3m
+	D = RSYSLOG3m
