diff --git a/krb/templates/etc/krb5.conf.erb b/krb/templates/etc/krb5.conf.erb
index 7ac603a..4a92ec8 100644
--- a/krb/templates/etc/krb5.conf.erb
+++ b/krb/templates/etc/krb5.conf.erb
@@ -1,5 +1,5 @@
 [libdefaults]
-	default_realm = RSYSLOG3
+	default_realm = RSYSLOG3h
 	kdc_timesync = 1
 	ccache_type = 4
 	forwardable = true
@@ -9,13 +9,13 @@
 	allow_weak_crypto = true
 
 [realms]
-	RSYSLOG3 = {
+	RSYSLOG3h = {
 		kdc = <%= @kdc_server_real %>
 		admin_server = <%= @kdc_server_real %>
-		default_domain = meta.zcu.cz
 	}
 
 [domain_realm]
-	.meta.zcu.cz = RSYSLOG3
-	meta.zcu.cz = RSYSLOG3
-	
+	A = RSYSLOG3h
+	B = RSYSLOG3h
+	C = RSYSLOG3m
+	D = RSYSLOG3m
