diff --git a/debian/changelog b/debian/changelog
index 154a4e06..bcff30fd 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+rsyslog (8.24.0.r50) unstable; urgency=medium
+
+  * Non maintainer upload
+  * metacentrum.cz specific changes
+
+ -- Radoslav Bodo <bodik@cesnet.cz>  Tue, 05 Oct 2017 23:23:23 +0200
+
 rsyslog (8.24.0-1) unstable; urgency=medium
 
   [ Michael Biebl ]
diff --git a/debian/control b/debian/control
index c72f3ee8..ad152517 100644
--- a/debian/control
+++ b/debian/control
@@ -1,7 +1,7 @@
 Source: rsyslog
 Section: admin
 Priority: important
-Maintainer: Michael Biebl <biebl@debian.org>
+Maintainer: Radoslav Bodo <bodik@cesnet.cz>
 Build-Depends: debhelper (>= 10),
                dpkg-dev (>= 1.17.14),
                dh-exec,
diff --git a/debian/rules b/debian/rules
index e3271480..57f340ff 100755
--- a/debian/rules
+++ b/debian/rules
@@ -46,7 +46,6 @@ override_dh_auto_configure:
 		--enable-omczmq \
 		--enable-omhiredis \
 		--disable-libgcrypt \
-		--enable-testbench \
 		--enable-imdiag \
 		--disable-generate-man-pages \
 		--with-systemdsystemunitdir=/lib/systemd/system
diff --git a/debian/source/format b/debian/source/format
index 163aaf8d..89ae9db8 100644
--- a/debian/source/format
+++ b/debian/source/format
@@ -1 +1 @@
-3.0 (quilt)
+3.0 (native)
