diff --git a/debian/changelog b/debian/changelog
index d06cfbb..3a03b2f 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,11 @@
-rsyslog (8.16.0-1~bpo8+1) jessie-backports; urgency=medium
+rsyslog (8.16.0~bpo8+1.rb40) jessie-backports; urgency=medium
+
+  * Non maintainer upload
+  * metacentrum.cz specific changes
+
+ -- Radoslav Bodo <bodik@cesnet.cz>  Tue, 05 Oct 2016 23:23:23 +0200
+
+syslog (8.16.0-1~bpo8+1) jessie-backports; urgency=medium
 
   * Rebuild for jessie-backports.
   * Revert "Enable CZMQ plugins", we don't have libczmq in jessie.
diff --git a/debian/control b/debian/control
index 081455f..d22a87a 100644
--- a/debian/control
+++ b/debian/control
@@ -1,7 +1,7 @@
 Source: rsyslog
 Section: admin
 Priority: important
-Maintainer: Michael Biebl <biebl@debian.org>
+Maintainer: Radoslav Bodo <bodik@cesnet.cz>
 Build-Depends: debhelper (>= 8),
                dpkg-dev (>= 1.16.1),
                dh-autoreconf,
diff --git a/debian/rules b/debian/rules
index b9df669..a1743dd 100755
--- a/debian/rules
+++ b/debian/rules
@@ -45,7 +45,6 @@ override_dh_auto_configure:
 		--enable-mmpstrucdata \
 		--enable-mmsequence \
 		--disable-libgcrypt \
-		--enable-testbench \
 		--enable-imdiag \
 		--disable-generate-man-pages \
 		--with-systemdsystemunitdir=/lib/systemd/system
diff --git a/debian/source/format b/debian/source/format
index 163aaf8..89ae9db 100644
--- a/debian/source/format
+++ b/debian/source/format
@@ -1 +1 @@
-3.0 (quilt)
+3.0 (native)
