commit edcaae1a526ae556bf431d7b482d1bb76820c8c5
Author: root <root@ztook5.meta.zcu.cz>
Date:   Thu Oct 5 16:26:50 2017 +0200

    meta-detachcfg.patch

diff --git a/debian/meta-remote.conf-DISABLED b/debian/meta-remote.conf-DISABLED
new file mode 100644
index 00000000..07e8fa51
--- /dev/null
+++ b/debian/meta-remote.conf-DISABLED
@@ -0,0 +1,25 @@
+#vzdalene logovani
+$ModLoad omgssapi
+$GSSForwardServiceName host
+$gssmode encryption
+
+$ActionQueueType LinkedList        # use asynchronous processing
+$ActionQueueFileName omgssapi1      # set file name, also enables disk mode
+$ActionResumeRetryCount -1         # infinite retries on insert failure
+$ActionQueueSaveOnShutdown on      # save in-memory data if rsyslog shuts down
+$ActionQueueMaxDiskSpace 100m      # limit disk cache
+$ActionQueueTimeoutEnqueue 100     # dont block worker indefinitely when cache fills up
+*.* :omgssapi:sysel.ics.muni.cz:515
+
+#module(load="omgssapi")
+#*.* action(type="omgssapi"
+#        target="<%= rsyslog_server %>"
+#        port="515"
+#        queue.FileName="omgssapi1"
+#        queue.Type="LinkedList"
+#        queue.MaxDiskSpace="100m"
+#        queue.SaveOnShutdown="on"
+#        queue.TimeoutEnqueue="100"
+#        action.ResumeRetryCount="-1"
+#)
+
diff --git a/debian/rsyslog.install b/debian/rsyslog.install
index 069e8e9f..434a6e7b 100755
--- a/debian/rsyslog.install
+++ b/debian/rsyslog.install
@@ -33,3 +33,4 @@ usr/lib/*/rsyslog/pm*.so
 [linux-any] usr/lib/*/rsyslog/imjournal.so
 [linux-any] usr/lib/*/rsyslog/omjournal.so
 lib/systemd/system/rsyslog.service
+debian/meta-remote.conf-DISABLED etc/rsyslog.d/
